.github/workflows/_ci.yaml

```diff
@@ -399,6 +399,180 @@ jobs:
         *-execution-combine.log
     secrets: inherit
 
+  test-nsys-jax-eks:
+    needs: build-jax
+    if: inputs.ARCHITECTURE == 'amd64' # arm64 runners n/a
+    runs-on: eks
+    env:
+      POST_PROCESS_JOB_DESCRIPTION: |
```

Comment:
How about saving the YAML specs as actual files under `.github/workflows`? You could also reuse something from https://github.com/jax-ml/jax/pull/24197/files although it is still not merged yet...

Reply:
We could do - the main counter-argument is that you then need some other solution for value substitution, whereas the status quo just uses GitHub Actions's substitution directly.

Reply:
Yes, either `envsubst` or `yq` should work. I find this template-substitution pattern to provide better readability and reusability in the long run.

Reply:
I reworked it with `yq`